nni/algorithms/compression/v2/pytorch/pruning/tools/task_generator.py

```diff
@@ -97,6 +100,9 @@ def generate_tasks(self, task_result: TaskResult) -> List[Task]:
     def generate_config_list(self, target_sparsity: List[Dict], iteration: int, compact2origin_sparsity: List[Dict]) -> List[Dict]:
         raise NotImplementedError()
 
+    def allocate_sparsity(self, new_config_list: List[Dict], model: Module, masks: Dict[str, Dict[str, Tensor]]):
+        return new_config_list
```

Comment:
why we need this function? directly return an input argument?

Reply:
Yes, this change is for AutoCompressTaskGenerator, it can apply `SimulatedAnnealingPruner.compress()` in this function.